ocs_ci/ocs/resources/pod.py

```diff
@@ -2187,18 +2187,26 @@ def run_osd_removal_job(osd_ids=None):
     ocp_version = get_ocp_version()
     ocs_version = config.ENV_DATA["ocs_version"]
 
-    if Version.coerce(ocs_version) >= Version.coerce(
-        "4.10"
-    ) and not check_safe_to_destroy_status(osd_ids_str):
-        cmd = f"process ocs-osd-removal -p FORCE_OSD_REMOVAL=true -p FAILED_OSD_IDS={osd_ids_str} -o yaml"
-    elif Version.coerce(ocp_version) >= Version.coerce("4.6"):
-        cmd = f"process ocs-osd-removal -p FAILED_OSD_IDS={osd_ids_str} -o yaml"
+    # Fixes: #6662
+    # Version OCS 4.6 and above requires FORCE_OSD_REMOVAL set to true in order to not get stuck
+    cmd_params = (
+        "-p FORCE_OSD_REMOVAL=true"
+        if Version.coerce(ocs_version) >= Version.coerce("4.6")
```

Comment:
When we are touching the code, can we maybe change to new approach for version comparison like here:

https://github.com/red-hat-storage/ocs-ci/blob/88a8b7918daf020181a93985645465b6a536f051/tests/ui/test_validation_ui.py#L38

Thanks